apps/dashboard/app/views/files/edit.html.erb

```diff
@@ -223,10 +223,18 @@
           </optgroup>
         </select>
       </li>
-      <li class="list-group-item hidden-xs px-2 form-group form-inline m-0 border-bottom-0">
+      <li class="list-group-item hidden-xs px-2 form-group form-inline m-0">
         <label class="control-label hidden-md hidden-lg navbar-text px-1" for="wordwrap">Wrap</label>
-        <div class="form-group">
-          <input type="checkbox" id="wordwrap" value="">
+        <div class="form-switch">
+          <input class="form-check-input" role="switch" type="checkbox" id="wordwrap" value="">
+        </div>
```

Comment:
Since it's only one other checkbox here I thought I may as well change it to a switch as well. Didn't want to initially, but seems small enough to smuggle in here.